README.md

```diff
@@ -156,6 +156,23 @@ $ bundle exec ruby benchmarks.rb
 If your results differ from what's posted here,
 [let us know](https://github.com/panorama-ed/memo_wise/issues/new)!
 
+## Thread Safety
+
+MemoWise makes the following **thread safety** guarantees on all supported Ruby
+versions:
+
+1. **Before** a value has been memoized
+
+   * Contended calls from multiple threads...
+      * May call original method multiple times
+      * May return different valid return values on different threads
```

Comment:
I worry people may not understand the context of this one. Maybe something like:

* May return different valid results (when the method is nondeterministic, like `rand`)